src/styles/social-card.css

```diff
@@ -0,0 +1,13 @@
+div.social-card {
+    width: 800px;
+    height: 418px;
+}
+
+.social-card h2 {
+    background-color: #ff5722;
+    color: white;
+}
+
+.social-card h3 {
+    font-size: 3rem;
+}
```

Comment:
Add a newline to end of files - it's kind of an old thing to work with automated file reading systems, but it's a good practice and you can see GitHub flags it up.